### PR TITLE
Fix/tao 6204/datatable pagination blocker

### PR DESCRIPTION
**Pull request by zagovorichev**

### CODE REVIEW

File: views/js/ui/pagination.js [hunk 1]
```diff
@@ -143,6 +138,14 @@ define(['jquery', 'lodash', 'i18n', 'ui/component', 'ui/pagination/paginationStr
                         provider.enableButton(provider.firstPageButton());
                     }
                 }
+            }
+        };
+
+        paginationComponent = component(pagination);
+
+        paginationComponent
+            .on('change', function() {
```
Comment (krampstudio) [inline views/js/ui/pagination.js]: You can group the 2 events: 
```
on('change enable', function(){
    this.refresh();
})   
```

File: views/js/test/ui/datatable/pagination/test.js [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+ *
+ * @author Oleksander Zagovorychev <olexander.zagovorychev@1pt.com>
+ */
+
+define([
+    'jquery',
+    'ui/datatable'
+], function($) {
+    "use strict";
+
```
Comment (krampstudio) [inline views/js/test/ui/datatable/pagination/test.js]: Please always add a 
```
QUnit.module('Pagination behavior');

QUnit.asyncTest('disable', ...
```

File: views/js/test/ui/datatable/pagination/test.js [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+    QUnit.asyncTest('Pagination disabled', function(assert){
+        QUnit.expect(6);
+
+        var $elt = $('#container-1');
+        assert.ok($elt.length === 1, 'Test the fixture is available');
+
+        $elt.on('create.datatable', function(){
+            assert.ok($elt.find('.datatable').length === 1, 'the layout has been inserted');
+            assert.ok($elt.find('.icon-backward').length === 1, 'there is 1 backward buttons');
+            assert.ok($elt.find('.icon-forward').length === 1, 'there is 1 forward buttons');
```
Comment (krampstudio) [inline views/js/test/ui/datatable/pagination/test.js]: You've a small typo : 1 button**s**

File: views/js/test/ui/datatable/pagination/test.js [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+define([
+    'jquery',
+    'ui/datatable'
+], function($) {
+    "use strict";
+
+    QUnit.asyncTest('Pagination disabled', function(assert){
+        QUnit.expect(6);
+
+        var $elt = $('#container-1');
```
Comment (krampstudio) [inline views/js/test/ui/datatable/pagination/test.js]: Please declare all variables at the top of the function to reflect hoisting : 

```
QUnit.asyncTest('Pagination disabled', function(assert){
    var $elt = $('#container-1');

    QUnit.expect(6);

```

Comment (krampstudio) [reply views/js/test/ui/datatable/pagination/test.js]: (and in other places too, see https://eslint.org/docs/rules/vars-on-top)

File: views/js/test/ui/datatable/pagination/test.js (hunk 3 above)
Comment (krampstudio) [reply views/js/test/ui/datatable/pagination/test.js]: see other places too

**Comment by zagovorichev** [comment]
@krampstudio Update

**Comment by krampstudio** [comment]
Last thing, the version bump is requested too

**Comment by krampstudio** [comment]
- [x] Code is now OK
 - [x] Unit tests run successfully
 - [x] No regression found in tao
 - [ ] Test the use in customer install

**Pull request merged**